examples/tpu_training-tf.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    # For simplicity, let's just tokenize our dataset as NumPy arrays 
</pre><p>What are other options here? Do you want to spend a sentence to make our users aware of that? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I added a comment that we discuss other options for creating the dataset below.</p>